style.module.scss

```diff
@@ -1,3 +1,82 @@
+.overviewInstallChoice, .overviewFeatureChoice {
+  margin-top: 15px;
+}
+.overviewInstallChoice + h2, .overviewFeatureChoice + h2 {
+  margin-top: 32px;
+}
+.overviewInstallChoiceStep, .overviewFeatureChoiceStep {
+  width: 90%;
+  display: flex;
+  justify-content: space-between;
+  align-items: left;
+  border: 1px solid black;
+  border-radius: 5px;
+  padding: 8px 10px 8px;
+  margin-bottom: 10px;
+  background-color: #f8f8f8;
+  border-color: #e8e8e8;
+  color: inherit;
+
+  &:hover {
+    color: inherit;
+    border-color: #e0e0e0;
+    background-color: #f0f0f0;
+  }
+
+  &:last-child {
+    margin-right: 0;
+  }
+
+  strong {
+    font-weight: 600;
+  }
+}
+
+.overviewInstallChoiceStep {
+  img {
+    max-width: 70%;
+    height: 55%;
+    margin: 0;
+    padding: 5px 0 5px 0;
+    border: none;
+    order: 1;
+  }
+
+  img, strong {
+    align-self: center;
+    text-align: center;
+  }
+}
+
+.overviewInstallChoiceStep {
+  height: 50px;
+}
+.overviewFeatureChoiceStep {
+  flex-direction: column;
+}
+
+@media (min-width: 620px) {
+  .overviewInstallChoiceStep img {
+    order: 0;
+  }
+  .overviewInstallChoice, .overviewFeatureChoice {
+    display: flex;
+    margin-bottom: 10px;
+  }
+  .overviewFeatureChoiceStep {
+    justify-content: space-between;
+    align-items: left;
+  }
+  .overviewInstallChoiceStep, .overviewFeatureChoiceStep {
+    flex-grow: 1;
+    width: 33%;
+    height: 80px;
```

Comment:
Setting a fixed height here means text can overflow out of the container:

![image](https://user-images.githubusercontent.com/159100/215546878-ec20d917-22c3-4bce-89da-ab3652460087.png)

If we want a bunch of aligned tiles, maybe `grid` would be a better mechanism? We should be able to do something like
```css
.overviewInstallChoiceStep, .overviewFeatureChoiceStep {
  display: grid;
  grid-template-columns: 1fr 1fr;
  gap: 20px;
}
```
to simplify this styling. (We can override `grid-template-columns` to one column for smaller viewports.) I can explore this if you'd like.

Reply:
Implemented, per your suggestion - thanks for the feedback!